app/javascript/components/chart/ChartResults.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="chart--results">
+    <svg width="100%" height="100%" :viewBox="`0 0 ${options.svgWidth} ${options.svgHeight}`" xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid">
```

Comment:
The percentage height doesn't seem to work properly in ie:
<img width="817" alt="screenshot 2019-03-04 at 15 18 35" src="https://user-images.githubusercontent.com/43963744/53742555-d1f1ac80-3e90-11e9-87ef-e14f576a5388.png">